specification/paths/Carriers-carrier_id-PickupDropoffLocations-country_code-postal_code.json

```diff
@@ -20,8 +20,8 @@
         ]
       }
     ],
-    "summary": "Get all the locations from a carrier.",
-    "description": "This endpoint retrieves pick-up and drop-off locations offered by the supplied carrier.<ul><li>Locations of the type `pick-up` can be used with services offering this as their `delivery_method`.</li><li>Locations of the type `drop-off` can be used with services offering this as their `handover_method`.</li></ul>",
+    "summary": "Get office locations from a carrier by country and postal code.",
+    "description": "This endpoint retrieves pick-up and drop-off locations by geolocation offered by the supplied carrier.<ul><li>Locations of the type `pick-up` can be used with services offering this as their `delivery_method`.</li><li>Locations of the type `drop-off` can be used with services offering this as their `handover_method`.</li></ul>",
```

Comment:
```suggestion
    "description": "This endpoint retrieves pick-up and drop-off locations by country code and postal code offered by the supplied carrier.<ul><li>Locations of the type `pick-up` can be used with services offering this as their `delivery_method`.</li><li>Locations of the type `drop-off` can be used with services offering this as their `handover_method`.</li></ul>",
```

Reply:
@yoan-myparcel Did you purposefully leave out the `radius` query parameter?

Reply:
Haha I have purposefully absolutely forgot about it 😆